app/views/cleaners/show.html.erb

```diff
@@ -1,19 +1,55 @@
-<p id="notice"><%= notice %></p>
+<div class="container">
+  <ol class="breadcrumb">
```

Comment:
@keyurgohil2426 I noticed that the `ol` and `if notice.present?` blocks are repeated in all the files. should be reusable.